src/sql/pool.rs

```diff
@@ -104,6 +110,15 @@ impl Pool {
             conn: Some(conn),
             _permit: permit,
         };
+        conn.pragma_update(
```

Comment:
Maybe reserve the only connection for writing and use other ones for reading instead?